aeron-samples/scripts/cluster/basic-auction-cluster

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+##
+## Copyright 2014-2020 Real Logic Limited.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+## https://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+
+source ./script-common
+
+mkdir -p logs
+
+# tag::start_jvm[]
+function startNode() {
+    ${JAVA_HOME}/bin/java \
+        --add-opens java.base/jdk.internal.ref=ALL-UNNAMED \
```

Comment:
I think this line should be `${ADD_OPENS} \` . I tried to run the script with JDK8 and failed at this line with `Unrecognized option: --add-opens`

Reply:
Cheers, yes that sample needs to be aware of the jdk version. Glad the tutorial was useful.

Reply:
That should be fixed now.